rules/windows/privilege_escalation_uac_bypass_diskcleanup_hijack.toml

```diff
@@ -12,21 +12,20 @@ stealthily execute code with elevated permissions.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "UAC Bypass via DiskCleanup Scheduled Task Hijack"
 risk_score = 47
 rule_id = "1dcc51f6-ba26-49e7-9ef4-2655abb2361e"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
 timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.args:(/autoclean or /AUTOCLEAN) and
-  process.parent.name:svchost.exe and
-  not process.executable:("C:\Windows\System32\cleanmgr.exe" or "C:\Windows\SysWOW64\cleanmgr.exe")
+process where event.type in ("start","process_started") and
+process.args:"/autoclean" and process.args:"/d" and
+not process.executable in ("C:\\Windows\\System32\\cleanmgr.exe","C:\\Windows\\SysWOW64\\cleanmgr.exe")
```

Comment:
do we want to add `process.parent.name:svchost.exe`? Otherwise, looks good!

Reply:
not necessary, avoid missing process.parent.name (observed in some instances its not captured on v7.10)

Reply:
if you think that removing the `process.parent.name`  won't cause FPs then this sounds good to me. i think your logic makes sense -- `process.args` is more than enough, so you don't gain much value by checking `process.parent.name` if it's redundant

if removing the `process.parent.name` field could introduce FPs, then we should leave the check and fix the bug in endpoint/beats instead

Reply:
its not in there (process.parent.name) and not necessary since process.args (autoclean + /d) is unique enough and in some instances (unrelated to this use case and especially for already running parent processes like services.exe, svchost.exe etc) I've observed that 7.10 is not capturing process.parent.name (only pid is captured).